docs/supported_versions.md

```diff
@@ -0,0 +1,206 @@
+            SRIOVFailedToConfigureVF URL: https://issues.redhat.com/browse/NHE-1171
+        reason: EvaluationFailed
+        status: Unknown
+        type: Recommended
+      release:
+        channels:
+        - candidate-4.16
+        - candidate-4.17
+        - eus-4.16
+        - fast-4.16
+        - stable-4.16
+        image: quay.io/openshift-release-dev/ocp-release@sha256:fb321a3f50596b43704dbbed2e51fdefd7a7fd488ee99655d03784d0cd02283f
+        url: https://access.redhat.com/errata/RHSA-2024:5107
+        version: 4.16.7
+      risks:
+      - matchingRules:
+        - promql:
+            promql: |
+              group(csv_succeeded{_id="d6d42268-7dff-4d37-92cf-691bd2d42f41", name=~"sriov-network-operator[.].*"})
+              or
+              0 * group(csv_count{_id="d6d42268-7dff-4d37-92cf-691bd2d42f41"})
+          type: PromQL
+        message: OCP Versions 4.14.34, 4.15.25, 4.16.7 and ALL subsequent versions
+          include kernel datastructure changes which are not compatible with older
+          versions of the SR-IOV operator. Please update SR-IOV operator to versions
+          dated 20240826 or newer before updating OCP.
+        name: SRIOVFailedToConfigureVF
+        url: https://issues.redhat.com/browse/NHE-1171
+```
+
+## Specifying the OCP release in HostedCluster Custom Resource
+`.spec.release` dictates the version of the control plane.
+
+The HostedCluster propagates the intended `.spec.release` to the `HostedControlPlane.spec.release` and runs the appropriate Control Plane Operator version.
+
+The HostedControlPlane orchestrates the rollout of the new version of the Control Plane components along with any OCP component in the data plane through the new version of the [cluster version operator (CVO)](https://github.com/openshift/cluster-version-operator). This includes resources like:
+
+- the CVO itself
+- cluster network operator (CNO)
+- cluster ingress operator
+- manifests for the kube API-server (KAS), scheduler, and manager
+- machine approver
+- autoscaler
+- infra resources needed to enable ingress for control plane endpoints (KAS, ignition, konnectivity, etc.)
+
+Traditionally, in standalone OCP, the CVO has been the sole source of truth for upgrades. In hosted OCP cluster, the responsibility is currently split between Control Plane Operator (CPO) and CVO. This enabled the flexibility and speed the HyperShift project needed for the CPO to support the management/guest cluster topology and the multiple customizations needed for manifests that otherwise would have needed to be segregated in the payload.
+
+Using the information from the `status.version.availableUpdates` and `status.version.conditionalUpdates` described in the previous section, you can set the `.spec.release` in the HostedCluster custom resource to start the control plane upgrade.
```

Comment:
> This enabled the flexibility and speed the HyperShift project needed for the CPO to support the management/guest cluster topology and the multiple customizations needed for manifests that otherwise would have needed to be segregated in the payload.

This can be dropped for these docs right?